tests/pages/externalpages/pocket/base.py

```diff
@@ -0,0 +1,101 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from pypom import Page, Region
+from selenium.webdriver.common.by import By
+
+
+class ScrollElementIntoView:
```

Comment:
copied from bedrock base in assumption it will be useful down the road, but maybe not worth including in this PR